test/Push.hs

```diff
@@ -22,37 +19,37 @@ import Network.Pusher
 
 test :: Spec
 test = do
-  describe "Interest names are validated" $ do
-    it "Correct Interest names are accepted" $
+  describe "Interest names" $ do
+    it "are accepted when using valid characters" $
```

Comment:
It's not a big deal, and you don't have to change this, but normally you would `describe` a thing that "does" something. So instead of `"Interest names"` it would be something like `"Interest names constructor"`, and then the `it` section reads better because the `it` becomes part of the sentence, so it would be `it "accepts valid characters"`.

Reply:
Noted thanks!